rest/available-phone-numbers/local-advanced-example-1/local-get-advanced-example-1.5.x.cs

```diff
@@ -0,0 +1,32 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using System.Collections.Generic;
+using Twilio.Rest.Api.V2010.Account.AvailablePhoneNumberCountry;
+using System.Linq;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        const string nearLatLong = "37.840699,-122.461853";
+        var localAvailableNumber = LocalResource.Read("US",
+                                                      nearLatLong: nearLatLong,
+                                                      distance: 50,
+                                                      contains: "555",
+                                                      inRegion: "CA");
+
+        var firstNumber = localAvailableNumber.FirstOrDefault();
+        if (firstNumber != null) {
```

Comment:
It's expected to put the open curly brace right after this line. This way is more Java-ish.